.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/node:7.10
```

Comment:
Does it matter which version of node we pick here?

Reply:
I haven't tested with other versions are you thinking earlier or later?

Reply:
I don't have any strong preference, just asking the question in case someone has a reason why we should use one version over another. Happy to leave as is!

Reply:
To be honest I just left the node version as the default. If anyone knows a reason to change it I'd be happy to.